### PR TITLE
Identify3 edit own profile

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] need sigid plumbing from core

### CODE REVIEW

**Comment by buoyad** [comment]
@keybase/react-hackers this is ready

File: shared/actions/json/profile.json [hunk 1]
```diff
@@ -4,25 +4,26 @@
     "import * as More from '../constants/types/more'"
   ],
   "actions": {
-    "addProof": { "platform": "More.PlatformsExpandedType" },
+    "addProof": {"platform": "More.PlatformsExpandedType"},
     "backToProfile": {},
     "cancelAddProof": {},
     "cancelPgpGen": {},
     "checkProof": {},
+    "checkProof2": {"sigID": "string"},
```
Comment (chrisnojima) [inline shared/actions/json/profile.json]: this isn't a replacement for checkproof so we should name it differently. maybe it should be like
checkBuildingProof and checkSingleProof or something

File: shared/util/feature-flags.desktop.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ const ff: FeatureFlags = {
   admin: false,
   chatIndexProfilingEnabled: false,
   foldersInProfileTab: false,
-  identify3: false,
+  identify3: true,
```
Comment (chrisnojima) [inline shared/util/feature-flags.desktop.js]: lets keep this admin

File: shared/tracker2/assertion/container.js [hunk 3]
```diff
@@ -1,25 +1,38 @@
...
+import * as RouteTreeGen from '../../actions/route-tree-gen'
 import * as WalletsType from '../../constants/types/wallets'
 import * as Constants from '../../constants/tracker2'
 import Assertion from '.'
 import openUrl from '../../util/open-url'
 
 type OwnProps = {|
+  isSuggestion?: boolean,
   username: string,
+  // allowMenu: boolean,
```
Comment (chrisnojima) [inline shared/tracker2/assertion/container.js]: 🔥

File: go/service/user.go [hunk 4]
```diff
@@ -475,6 +475,8 @@ func (h *UserHandler) proofSuggestionsHelper(mctx libkb.MetaContext) (ret []Proo
 		switch displayConfig.Key {
 		case "zcash.t", "zcash.z", "zcash.s":
 			altKey = "zcash"
+		case "bitcoin":
+			altKey = "btc"
```
Comment (chrisnojima) [inline go/service/user.go]: this this affect ident2?

File: shared/util/feature-flags.desktop.js (hunk 2 above)
Comment (buoyad) [reply shared/util/feature-flags.desktop.js]: Oops thanks

File: go/service/user.go (hunk 4 above)
Comment (mlsteele) [reply go/service/user.go]: Nope this only affects the ProofSuggestions RPC which was not used prior to id3

File: shared/actions/profile/proofs.js [hunk 5]
```diff
@@ -38,6 +41,18 @@ const checkProof = (state, action) => {
     })
 }
 
+const recheckProof = (state, action) =>
+  flags.identify3 &&
+  RPCTypes.proveCheckProofRpcPromise({sigID: action.payload.sigID}, Constants.waitingKey).then(() =>
+    Tracker2Gen.createLoad({
```
Comment (buoyad) [inline shared/actions/profile/proofs.js]: @chrisnojima this seems heavy handed, does it seem right to you? I was looking for something that triggers `identify3UpdateRow`

Comment (chrisnojima) [reply shared/actions/profile/proofs.js]: lets keep it simple, it should be fast as everything should be cached

**Comment by chrisnojima** [comment]
👍

**Pull request merged**